NExtends/Expressions/ExpressionMerger.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Linq.Expressions;
+
+namespace NExtends.Expressions
+{
+	public class ExpressionMerger : ExpressionVisitor
+	{
+		Expression CurrentParameterExpression { get; set; }
+
+		public static Expression<Func<TIn, TOut>> Merge<TIn, TA, TOut>(Expression<Func<TIn, TA>> entryPoint, Expression<Func<TA, TOut>> expression1)
+		{
+			return new ExpressionMerger().MergeAll<TIn, TA, TOut>(entryPoint, expression1);
+		}
+
+		//possibilité de merger n expressions
+		//public static Expression<Func<TIn, TOut>> Merge<TIn, TA, TB, TOut>(Expression<Func<TIn, TA>> entryPoint, Expression<Func<TA, TB>> expression1, Expression<Func<TB, TOut>> expression2)
+		//{
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS125)